lib/game_team.rb

```diff
@@ -0,0 +1,35 @@
+class GameTeam
+  attr_reader :game_id, 
+              :team_id, 
+              :hoa, 
+              :result, 
+              :settled_in, 
+              :head_coach, 
+              :goals, 
+              :shots, 
+              :tackles,
+              :pim,
+              :powerplayopportunities,
+              :powerplaygoals,
+              :faceoffwinpercentage,
+              :giveaways,
+              :takeaways
+
+  def initialize(details)
+    @game_id = details[:game_id]
+    @team_id  = details[:team_id]
+    @hoa = details[:hoa]
+    @result = details[:result]
+    @settled_in = details[:settled_in]
+    @head_coach = details[:head_coach]
+    @goals = details[:goals]
+    @shots = details[:shots]
+    @tackles = details[:tackles]
+    @pim = details[:pim]
+    @powerplayopportunities = details[:powerplayopportunities]
```

Comment:
We may want to change variables like this one to 'power_play_opportunities', just to follow ruby convention. We can talk about if that's actually worth it or not but that's what I did with teams.